ports/geckolib/glue.rs

```diff
@@ -356,7 +356,7 @@ pub extern "C" fn Servo_StyleSet_Drop(data: *mut RawServoStyleSet) -> () {
 }
 
 pub struct GeckoDeclarationBlock {
-    pub declarations: Option<PropertyDeclarationBlock>,
+    pub declarations: Option<Arc<PropertyDeclarationBlock>>,
```

Comment:
I don't think so, I think the double indirection is fine for now. If for some reason it becomes a perf issue we can try to be a bit smarter, maybe conditionally compiling the extra gecko fields inside `PropertyDeclarationBlock`, or similar.